packages/esm-ward-app/src/hooks/useObs.ts

```diff
@@ -0,0 +1,22 @@
+import useSWR from 'swr';
+import { type Observation } from '../types';
+import { type Link, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+
+type QueueEntrySearchCriteria = Record<'patient', string> | Record<'encounter', string> | Record<'q', string>;
```

Comment:
What is this QueueEntrySearchCriteria here for?

Reply:
copy pasted error. Should have been `ObsSearchCriteria`